lib/SPIRV/libSPIRV/SPIRVModule.cpp

```diff
@@ -129,15 +129,15 @@ class SPIRVModuleImpl : public SPIRVModule {
   SPIRVMemoryModelKind getMemoryModel() const override { return MemoryModel; }
   SPIRVConstant *getLiteralAsConstant(unsigned Literal) override;
   unsigned getNumEntryPoints(SPIRVExecutionModelKind EM) const override {
-    auto Loc = EntryPointVec.find(EM);
-    if (Loc == EntryPointVec.end())
+    auto Loc = EntryPointMapVec.find(EM);
+    if (Loc == EntryPointMapVec.end())
       return 0;
     return Loc->second.size();
   }
   SPIRVFunction *getEntryPoint(SPIRVExecutionModelKind EM,
                                unsigned I) const override {
-    auto Loc = EntryPointVec.find(EM);
-    if (Loc == EntryPointVec.end())
+    auto Loc = EntryPointMapVec.find(EM);
+    if (Loc == EntryPointMapVec.end())
       return nullptr;
     assert(I < Loc->second.size());
     return get<SPIRVFunction>(Loc->second[I]);
```

Comment:
AFAIK there are no usages for these 2 methods. Perhaps we can remove them.
Also I'm not sure we need `EntryPointMapVec` at all.